osu.Game/Screens/Play/HUD/HitErrorDisplay.cs

```diff
@@ -116,12 +153,21 @@ private void createColour(bool rightAligned)
                 Alpha = 0,
             };
 
-            completeDisplayLoading(display);
+            completeDisplayLoading(display, rightAligned);
+        }
+
+        private void createCombined(bool rightAligned)
+        {
+            createBar(rightAligned);
+            createColour(rightAligned);
         }
 
-        private void completeDisplayLoading(HitErrorMeter display)
+        private void completeDisplayLoading(HitErrorMeter display, bool rightAligned)
```

Comment:
`rightAligned` seems not too future-proof seeing that there are already requests for the meters to come back as bottom-center aligned (#5932). Enum would be more readable, too.